src/client/LanguageClient.ts

```diff
@@ -344,9 +344,35 @@ export async function activate(context: ExtensionContext) {
   await ensureBundledDsReadOnly();
   await yieldToNextMacroTask();
 
-  client.outputChannel.appendLine('MetaEd has started 🎬');
-  // eslint-disable-next-line no-void
-  void window.showInformationMessage('MetaEd has started 🎬');
+  client.outputChannel.appendLine('MetaEd has started 🎬 ');
+
+  if (!isDataStandardProjectInWorkspace()) {
```

Comment:
So, this handles the use case of extension authors really well, which is what the ticket is about. However, it won't be as good an experience for the DS team. They don't use the bundled DS projects but instead operate on a DS project from a git repo.

I think the easy answer is to just skip this if the Alliance Mode flag is turned on in settings.